src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets

```diff
@@ -207,6 +198,18 @@ Copyright (c) .NET Foundation. All rights reserved.
           ItemName="_SourceFiles" />
     </MSBuild>
   </Target>
+  
+  <!--Projects with target framework like UWP, Win8, wpa81 produce a Pri file
+    in their bin dir. This Pri file is not included in the BuiltProjectGroupOutput, and
+    has to be added manually here.-->
+  <Target Name="_AddPriFileToPackBuildOutput"
```

Comment:
Looks like you could condition this target on IncludeProjectPriFile and checking if ProjectPriFullPath is non-empty. Not sure it would actually save much for perf however.